server/chat-plugins/mafia.ts

```diff
@@ -2500,6 +2522,25 @@ export const commands: ChatCommands = {
 			`/mafia spiritstump [player] Kills a player, but allows them to talk during the day, and vote on the lynch.`,
 		],
 
+		revealrole(target, room, user) {
```

Comment:
this needs to be tagged as a global command for the htmlroom buttons to work (`'!revealrole': true`)